crates/jstz_node/openapi.json

```diff
@@ -14,6 +14,201 @@
     "version": "0.1.0-alpha.0"
   },
   "paths": {
+    "/accounts/{address}/balance": {
+      "get": {
+        "tags": [
+          "Accounts"
+        ],
+        "summary": "Get balance of an account",
+        "operationId": "get_balance",
+        "parameters": [
+          {
+            "name": "address",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "text/plain": {
+                "schema": {
+                  "type": "integer",
+                  "format": "int64",
+                  "minimum": 0
+                }
+              }
+            }
+          },
+          "404": {
+            "description": ""
+          },
+          "500": {
+            "description": ""
+          }
+        }
+      }
+    },
+    "/accounts/{address}/code": {
+      "get": {
+        "tags": [
+          "Accounts"
+        ],
+        "summary": "Get code of an account",
+        "operationId": "get_code",
+        "parameters": [
+          {
+            "name": "address",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ParsedCode"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": ""
+          },
+          "404": {
+            "description": ""
+          },
+          "500": {
+            "description": ""
+          }
+        }
+      }
+    },
+    "/accounts/{address}/kv": {
+      "get": {
+        "tags": [
+          "Accounts"
+        ],
+        "summary": "Get KV value under a given key path",
+        "description": "Get KV value under a given key path for an account. If `key` is not provided,\nthe empty key path will be used.",
```

Comment:
It seems that multiline comments are not concatenated and we have a `\n` here. Not sure if there's any config for this.

Reply:
Yeah not sure we could get a round that but it generates ok on Scalar and, hopefully, client libs.